### PR TITLE
Timer Widget

### PR DESCRIPTION
**Pull request by HenderikusP**
**Issue:**
[Issue 65](https://github.com/SE701Group2/daily-focus/issues/65)

**Solution:**
A stop watch counter was made following the design found here [Figma](https://www.figma.com/file/J2DDVa9redQigCzTee56xw/SE701-Daily-Focus?node-id=0%3A1)

**Test Method:** 
will create a follow up PR if needed.

**Risk:**
If the method of incrementing the time is not accurate enough, with long time recordings it may be inaccurate.

**Reviewed By:**

### CODE REVIEW

File: client/src/components/Timer/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React, { useEffect, useState } from "react";
+import { Card, CardHeader, MenuItem, Select } from "@material-ui/core";
+import Button from "@material-ui/core/Button";
+import { IconButton, makeStyles } from "@material-ui/core";
+import styles from "./styles.module.scss";
```
Comment (Denijar) [inline client/src/components/Timer/index.js:5]: filename should be `style.module.scss`

File: client/src/components/Timer/styles.module.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    border-radius: 25px;
+    box-shadow: 2px 4px 31px 13px rgba(0, 0, 0, 0.1);
+}
+
+.buttonContainer {
+    display: inline-flex;
+    margin-bottom: 25px;
+}
+
+.timer_title {
```
Comment (Denijar) [inline client/src/components/Timer/styles.module.scss:18]: would be good to stay consistent with using camel case names

File: client/src/components/WidgetIcon/style.module.scss [hunk 3]
```diff
@@ -1,25 +0,0 @@
-.card {
```
Comment (Denijar) [inline client/src/components/WidgetIcon/style.module.scss:1]: Why did you delete this file?

File: client/src/App.js [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
 
 import Header from "./components/Header";
+import Timer from "./components/Timer";
 import "./App.css";
 
 function App() {
     return (
         <div className="App">
             <Header />
+            <Timer />
```
Comment (Denijar) [inline client/src/App.js:11]: To make sure that we don't have conflicts with PRs, it is easier if there are no changes in the App.js file. @Chuppay will handle rendering the component in issue #81

**Review by Denijar — CHANGES_REQUESTED** [review]
My comments are mostly consistency nits. The component styling looks great, nice job.

File: client/src/components/Timer/index.js (hunk 1 above)
Comment (HenderikusP) [reply client/src/components/Timer/index.js:5]: hahaha okie np :)

**Review by dikunong — APPROVED** [review]
LGTM 😎

**Review by Denijar — APPROVED** [review]
LGTM good job :)
